examples/test_cq.rs

```diff
@@ -0,0 +1,25 @@
+use sideway::verbs::comp_channel;
+use sideway::verbs::completion_queue;
+use sideway::verbs::device;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let device_list = device::DeviceList::new()?;
+    for device in &device_list {
+        let ctx = device.open().unwrap();
+
+        let comp_channel = comp_channel::CompletionChannel::new(&ctx).unwrap();
+        let mut builder = completion_queue::CompletionQueueBuilder::new(&ctx);
```

Comment:
What about using `ctx.create_comp_channel` `ctx.create_cq_builder`? So that we don't need to pass the argument explicitly.

Reply:
Yeah we can add some methods at context level to invoke these implementations.